runtime/src/tests.rs

```diff
@@ -130,6 +138,156 @@ fn new_test_ext() -> sp_io::TestExternalities {
     ext
 }
 
+fn call_transfer(dest: AccountId32, value: u128) -> Box<RuntimeCall> {
+    Box::new(RuntimeCall::Balances(BalancesCall::transfer_allow_death { dest: MultiAddress::Id(dest), value }))
+}
+#[test]
+fn test_multisig_send_and_return() {
+    new_test_ext().execute_with(|| {
+        // Extract the account IDs from SAMPLE_USERS
+        let account_ids: Vec<AccountId32> = testsfixtures::SAMPLE_USERS.iter().map(|user| AccountId32::from(user.raw_account)).collect();
+
+        let multi = Multisig::multi_account_id(&account_ids[..3], 2);
+
+        assert_ok!(Balances::transfer_allow_death(RuntimeOrigin::signed(account_ids[0].clone()), MultiAddress::Id(multi.clone()), 500));
+        assert_ok!(Balances::transfer_allow_death(RuntimeOrigin::signed(account_ids[1].clone()), MultiAddress::Id(multi.clone()), 500));
+        assert_ok!(Balances::transfer_allow_death(RuntimeOrigin::signed(account_ids[2].clone()), MultiAddress::Id(multi.clone()), 500));
+
+        // Balance for account 0 reduced by 500, after it was sent to the multisig
+        assert_eq!(Balances::free_balance(account_ids[0].clone()), testsfixtures::SAMPLE_USERS[0].starting_balance - 500);
```

Comment:
You did a great job on writing these explorative tests that are useful to understand how the pallet works but here you're essentially testing substrate's code. I'm incline to trust substrate code and write just tests that trying to verify almost all possible the code that we wrote.

My idea is to have essentially the following tests:

1. `pallet_multisig_availability()`: just check you can use this pallet by call any kind of available extrinsic on this pallet - if this test compile you have a proof that you have had added this feature at your chain
2. `use_correct_weights::pallet_multisig()`: you already wrote 
3. `pallets_interact::sudo_multisig`: just move `test_set_sudo_to_multisig()` in `pallets_interact` module and change name; that's exactly a bussines case that we need to check and have a test that is here in place to attesting this case is useful.
4. (_Not sure about this test_) `pallet_multisig_configuration()`: here you should test that you configured the pallet correctly with some bussines cases.... the hard part here is what're the bussines cases maybe the deposit should be enought to cover the storage cost but what we are testing? that the constants are equals to some other constants?... Maybe that's really useless.

So maybe is better to remove this and the follow test... :cry:

Reply:
`pallet_multisig_availability` -> actually that's exactly my first test, I will keep that one as that one uses an extrinsic in the pallet

`use_correct_weights::pallet_multisig()` -> that one is done as you said

`pallets_interact::sudo_multisig` -> I'll move this one to pallets_interact as requested to a new module called sudo

I'll remove `cancel_multisig_returns_deposit` 

About `pallet_multisig_configuration` yeah I think we can skip it for now.